res/values/strings.xml

```diff
@@ -125,6 +125,9 @@
   <string name="ui_mark_as_read_label">Mark as read (emails)</string>
   <string name="ui_mark_as_read_desc">Whether to mark messages in Gmail as read or not.</string>
 
+  <string name="ui_sync_read_flag_label">Save read flag</string>
+  <string name="ui_sync_read_flag_desc">Whether to mark unread SMS and MMS in Gmail as unread or not.</string>
```

Comment:
the wording is very confusing here, not sure anyone would understand the difference between "Mark as read" and  "Save read flag"